common/e2e_agent/client.go

```diff
@@ -1036,3 +1038,53 @@ func ZfsDestroyPool(serverAddr string, poolName string) (string, error) {
 	logf.Log.Info("ZfsDestroyPool succeeded", "output", out)
 	return out, err
 }
+
+// PrepareHostPathDisk create hostapth disk
```

Comment:
"PrepareHostPathDisk creates disk of type hostPath".  Could this be CreateHostPathDisk?

Reply:
[Done](https://github.com/openebs/openebs-e2e/compare/a7bfee69eace1eda08b27a2f9c0061fdf39fab99..89156f9c0f293a6f8cf2808b7806910cb2b5c6d5)